### ISSUE TITLE
Question : Can I install and test mailinbox on my local dev machine before installing on remote VPS?

### ISSUE THREAD

**Issue by webb24h** [mail-in-a-box/mailinabox#2117]
Hi,

So I have a VPS running Ubuntu server for our company custom domain. As most devs do, we test everything locally before deploying remotely. I'm running Ubuntu and Node on my local dev. Can mailinbox be tested locally like you would test Node locally or does it only work with public DNS?

**Comment by i0t4** [comment]
no, your isp would block the mail to any email addresses besides localhost
